code/modules/robotics/bot/secbot.dm

```diff
@@ -1,3 +1,9 @@
+#define IS_NOT_BEEPSKY_AND_HAS_SOME_GENERIC_BATON 0				// Just some everyday bot on the beat
+#define IS_BEEPSKY_AND_HAS_HIS_SPECIAL_BATON 1						// Full-assed Beepsky
+#define IS_NOT_BEEPSKY_BUT_HAS_HIS_SPECIAL_BATON 2				// A Beepsky brand secboton
+#define IS_BEEPSKY_BUT_HAS_SOME_GENERIC_BATON 3						// A generic-ass shitcurity baton
+#define IS_NOT_BEEPSKY_BUT_HAS_A_GENERIC_SPECIAL_BATON 4	// A generic, non-Beepsky brand secboton
```

Comment:
I feel like this sort of thing would be better either as two variables (is_beepsky, baton_type) or through bit flags in one variable but it's ok even like this to some extent.

Reply:
I don't really know how bitflags work, when I do, I'll definitely think about using them!